sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2812,4 +2813,18 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
         "location" -> toSQLValue(location.toString, StringType),
         "identifier" -> toSQLId(tableId.nameParts)))
   }
+
+  def cannotConvertCatalystValueToProtobufEnumTypeError(
+      sqlColumn: Seq[String],
+      protobufColumn: String,
+      data: String,
+      enumString: String): Throwable = {
+    new AnalysisException(
+      errorClass = "CANNOT_CONVERT_SQL_VALUE_TO_PROTOBUF_ENUM_TYPE",
```

Comment:
@justaparth we forgot to update core/src/main/resources/error/error-classes.json and docs/sql-error-conditions.md in this PR.
Don't know how this got merged without failing. 
Could you send a follow up PR updating those files? Also, it will be better if we verify this error with `checkError()` in unit tests (see similar examples).